src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    private static final Random RANDOM = new Random();
+
+    public static Colors getRandomColor() {
```

Comment:
```suggestion
    public Color getRandomColor() {
```
create an instance of this class in Lottery and use its method